src/main/shared-cre-setup.js

```diff
@@ -567,9 +567,7 @@ function calcCR(effectiveness, trapPower, trapLuck, mousePower) {
  */
 function minLuck(effectiveness, mousePower) {
   var finalEffectiveness = Math.min(effectiveness, 2);
-  var minLuckSquared =
-    mousePower / (3 - finalEffectiveness) / Math.pow(finalEffectiveness, 2);
-  return Math.ceil(Math.sqrt(minLuckSquared));
+  return Math.ceil(Math.ceil(Math.sqrt(finalEffectiveness/2)) / Math.min(1.4, effectiveness));
```

Comment:
Why doesn't this version of `minLuck` depend on `mousePower`?

Reply:
Probably a typo ;)